[2;35m2017-09-12 01:45:11,844 - WARN  [Thread-42:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :34, action:com.klw.fastfun.pay.common.action.CommonAction, method:getProvinceByPhone
[m[2;35m2017-09-12 01:45:11,880 - WARN  [Thread-82:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :26, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 01:45:11,898 - WARN  [Thread-157:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :10, action:com.klw.fastfun.pay.common.action.CommonAction, method:querySmsFilterBySpid
[m[2;35m2017-09-12 01:45:17,005 - WARN  [Thread-52:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :7, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 01:45:17,022 - WARN  [Thread-113:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :11, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-09-12 01:45:33,037 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x58449360, /127.0.0.1:54679 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 01:45:33,037 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x9b083087, /127.0.0.1:54677 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 01:45:33,037 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x636ff349, /127.0.0.1:54678 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-09-12 01:46:04,948 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-09-12 01:46:05,453 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-09-12 01:46:05,453 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-09-12 01:46:05,454 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-09-12 01:46:05,670 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-09-12 01:46:05,691 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-09-12 01:46:05,700 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-09-12 01:46:05,706 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-09-12 01:46:05,706 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-09-12 01:46:05,716 - INFO  [main:ServerBootstrap@129] - Application PID:	9472
[m[7;31m2017-09-12 01:46:06,375 - ERROR [Xmemcached-Reactor-2:MemcachedHandler@138] - XMemcached network layout exception
[mjava.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at com.google.code.yanf4j.nio.impl.NioTCPSession.readFromBuffer(NioTCPSession.java:203)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onRead(AbstractNioSession.java:196)
	at com.google.code.yanf4j.nio.impl.AbstractNioSession.onEvent(AbstractNioSession.java:341)
	at com.google.code.yanf4j.nio.impl.SocketChannelController.dispatchReadEvent(SocketChannelController.java:56)
	at com.google.code.yanf4j.nio.impl.NioController.onRead(NioController.java:157)
	at com.google.code.yanf4j.nio.impl.Reactor.dispatchEvent(Reactor.java:295)
	at com.google.code.yanf4j.nio.impl.Reactor.run(Reactor.java:142)
[2;35m2017-09-12 01:46:06,387 - WARN  [Xmemcached-Reactor-2:MemcachedConnector@365] - Remove a session: 127.0.0.1:11211
[m[2;35m2017-09-12 01:46:08,389 - WARN  [Heal-Session-Thread:MemcachedConnector$SessionMonitor@125] - Trying to connect to 127.0.0.1:11211 for 1 times
[m[2;35m2017-09-12 01:46:08,389 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;35m2017-09-12 01:46:23,195 - WARN  [Thread-153:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :34, action:com.klw.fastfun.pay.common.action.CommonAction, method:getProvinceByPhone
[m[2;35m2017-09-12 01:46:23,231 - WARN  [Thread-42:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :27, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 01:46:23,250 - WARN  [Thread-122:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :12, action:com.klw.fastfun.pay.common.action.CommonAction, method:querySmsFilterBySpid
[m[2;35m2017-09-12 01:49:56,103 - WARN  [Thread-132:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :7, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 01:49:56,125 - WARN  [Thread-2:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :15, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-09-12 01:49:59,130 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x0e6e04e5, /127.0.0.1:54984 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 01:49:59,132 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xd2e97f07, /127.0.0.1:54986 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 01:49:59,132 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x7fee4c6c, /127.0.0.1:54985 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-09-12 10:54:18,398 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-09-12 10:54:19,117 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-09-12 10:54:19,117 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-09-12 10:54:19,117 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-09-12 10:54:19,412 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-09-12 10:54:19,431 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-09-12 10:54:19,440 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-09-12 10:54:19,447 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-09-12 10:54:19,447 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-09-12 10:54:19,456 - INFO  [main:ServerBootstrap@129] - Application PID:	8640
[m[2;35m2017-09-12 10:54:26,040 - WARN  [Thread-12:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :35, action:com.klw.fastfun.pay.common.action.CommonAction, method:getProvinceByPhone
[m[2;35m2017-09-12 10:54:26,079 - WARN  [Thread-94:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :29, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 10:54:26,099 - WARN  [Thread-103:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :12, action:com.klw.fastfun.pay.common.action.CommonAction, method:querySmsFilterBySpid
[m[2;35m2017-09-12 10:54:42,363 - WARN  [Thread-43:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :10, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[2;32m2017-09-12 11:02:54,492 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-09-12 11:02:54,917 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-09-12 11:02:54,917 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-09-12 11:02:54,917 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-09-12 11:02:55,125 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-09-12 11:02:55,146 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-09-12 11:02:55,155 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-09-12 11:02:55,162 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-09-12 11:02:55,163 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-09-12 11:02:55,172 - INFO  [main:ServerBootstrap@129] - Application PID:	6696
[m[2;35m2017-09-12 11:03:05,874 - WARN  [Thread-152:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :18, action:com.klw.fastfun.pay.common.action.CommonAction, method:getProvinceByPhone
[m[2;35m2017-09-12 11:03:05,894 - WARN  [Thread-92:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :9, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 11:03:05,904 - WARN  [Thread-76:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :3, action:com.klw.fastfun.pay.common.action.CommonAction, method:querySmsFilterBySpid
[m[2;35m2017-09-12 11:03:14,301 - WARN  [Thread-153:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :15, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-09-12 11:05:09,763 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xc51b3378, /127.0.0.1:50065 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 11:05:09,765 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xe82553b4, /127.0.0.1:50064 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 11:05:09,774 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xcfaaadfc, /127.0.0.1:50066 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-09-12 21:40:35,197 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-09-12 21:40:35,602 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-09-12 21:40:35,602 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-09-12 21:40:35,602 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-09-12 21:40:35,836 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-09-12 21:40:35,852 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-09-12 21:40:35,852 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-09-12 21:40:35,867 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-09-12 21:40:35,867 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-09-12 21:40:35,867 - INFO  [main:ServerBootstrap@129] - Application PID:	9408
[m[2;35m2017-09-12 21:40:39,085 - WARN  [Thread-62:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :31, action:com.klw.fastfun.pay.common.action.CommonAction, method:getProvinceByPhone
[m[2;35m2017-09-12 21:40:39,117 - WARN  [Thread-142:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :16, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 21:40:39,148 - WARN  [Thread-22:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :15, action:com.klw.fastfun.pay.common.action.CommonAction, method:querySmsFilterBySpid
[m[2;35m2017-09-12 21:41:05,194 - WARN  [Thread-112:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :16, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 21:41:05,211 - WARN  [Thread-52:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :17, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-09-12 21:41:12,826 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xa93384e6, /127.0.0.1:56493 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 21:41:12,842 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x105ecdf2, /127.0.0.1:56494 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 21:41:12,826 - ERROR [JUICE-SERVER-Handler-4-1-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x8437ec7d, /127.0.0.1:56495 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[2;32m2017-09-12 21:42:29,124 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-09-12 21:42:29,779 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-09-12 21:42:29,779 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-09-12 21:42:29,779 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-09-12 21:42:29,966 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-09-12 21:42:29,997 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-09-12 21:42:29,998 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-09-12 21:42:29,998 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-09-12 21:42:30,014 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-09-12 21:42:30,014 - INFO  [main:ServerBootstrap@129] - Application PID:	6968
[m[2;32m2017-09-12 21:57:06,064 - INFO  [main:ServerBootstrap@78] - connect to database:jdbc:mysql://119.23.172.103:3306/actual?useUnicode=true&characterEncoding=UTF-8
[m[2;32m2017-09-12 21:57:06,485 - INFO  [main:ServerBootstrap@104] - Configuration Server bootstrap params.......
[m[2;32m2017-09-12 21:57:06,485 - INFO  [main:ServerBootstrap@105] - Server protocol:	socket
[m[2;32m2017-09-12 21:57:06,485 - INFO  [main:ServerBootstrap@106] - Server port:	9912
[m[2;35m2017-09-12 21:57:06,745 - WARN  [main:XMemcachedClient@727] - XMemcachedClient use Binary protocol
[m[2;32m2017-09-12 21:57:06,765 - INFO  [main:SelectorManager@37] - Creating 16 reactors...
[m[2;35m2017-09-12 21:57:06,775 - WARN  [main:AbstractController@372] - The Controller started at localhost/127.0.0.1:0 ...
[m[2;35m2017-09-12 21:57:06,785 - WARN  [Xmemcached-Reactor-0:MemcachedConnector@251] - Add a session: 127.0.0.1:11211
[m[2;32m2017-09-12 21:57:06,785 - INFO  [main:Application@25] - 初始化AO...
[m[2;32m2017-09-12 21:57:06,785 - INFO  [main:ServerBootstrap@129] - Application PID:	7044
[m[2;35m2017-09-12 21:57:22,805 - WARN  [Thread-142:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :20, action:com.klw.fastfun.pay.common.action.CommonAction, method:getProvinceByPhone
[m[2;35m2017-09-12 21:57:22,826 - WARN  [Thread-52:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :11, action:com.klw.fastfun.pay.common.action.CommonAction, method:getSdkPayCodes
[m[2;35m2017-09-12 21:57:22,836 - WARN  [Thread-22:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :10, action:com.klw.fastfun.pay.common.action.CommonAction, method:querySmsFilterBySpid
[m[2;35m2017-09-12 21:57:59,934 - WARN  [Thread-92:JuiceThreadPool$JuiceWorkThread@108] - RPC method start time :10, action:com.klw.fastfun.pay.common.action.CommonAction, method:addOrderReqInfo
[m[7;31m2017-09-12 21:58:13,021 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x74d38428, /127.0.0.1:57389 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 21:58:13,021 - ERROR [JUICE-SERVER-Handler-4-2-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0xe618412c, /127.0.0.1:57390 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
[7;31m2017-09-12 21:58:13,021 - ERROR [JUICE-SERVER-Handler-4-3-thread:ConnectionException@22] - uncaughtException :
[mcom.juice.orange.game.exception.JuiceException: 远程主机强迫关闭了一个现有的连接。 on [id: 0x200f82c1, /127.0.0.1:57388 => /127.0.0.1:9912]
	at com.juice.orange.game.exception.JuiceException.fromException(JuiceException.java:38)
	at com.juice.orange.game.exception.JuiceException.fromExceptionEvent(JuiceException.java:32)
	at com.juice.orange.game.util.ConnectionHelper$2.run(ConnectionHelper.java:56)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: 远程主机强迫关闭了一个现有的连接。
	at sun.nio.ch.SocketDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(Unknown Source)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.read(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source)
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:107)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:88)
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	... 3 more
